python/pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "pyarrow>=4",
     'numpy<1.20.0;python_version<="3.6"',
     'dataclasses;python_version<="3.6"',
+    "packaging>=20",
```

Comment:
Just my 2 cents, maybe not worth introducing an extra dependency if all we need is parsing pyarrow major version number. Would be good to keep the runtime dependency as lean as possible.

Reply:
Hmm yeah I guess I can do a string split to get the major version.

Reply:
It's a dependency of pytest, so we already have it implicitly in our dev environment FYI. So I'll leave it in the tests.